encord/storage.py

```diff
@@ -57,41 +57,98 @@
 
 
 class StorageFolder:
+    """
+    Represents a storage folder within the Encord system.
+
+    Attributes:
+        api_client (ApiClient): The API client used for making requests.
```

Comment:
Private attributes that I wouldn't really document (Note _ prefix). Not actually used by the enduser

Reply:
It looks like you have removed all strings for private **Methods** not attributes. Not necessarily bad but other way round.